app/components/anchor/logo_component.rb

```diff
@@ -0,0 +1,45 @@
+module Anchor
+  class LogoComponent < Component
+    VARIANT_DEFAULT = :default
+    VARIANT_MAPPINGS = {
+      VARIANT_DEFAULT => "buoy-logo",
+      :mark => "buoy-logomark",
+      :stacked => "buoy-logo-stacked",
+    }.freeze
+    VARIANT_OPTIONS = VARIANT_MAPPINGS.keys
+
+    VARIANT_ASPECT_RATIO_MAPPINGS = {
+      default: "500 / 81",
+      mark: "72 / 58",
+      stacked: "500 / 341",
+    }.freeze
```

Comment:
What you think about mapping each aspect ratio variant to a static tailwind class?
```suggestion
    VARIANT_ASPECT_RATIO_MAPPINGS = {
      default: "aspect-[500 / 81]",
      mark: "aspect-[72 / 58]",
      stacked: "aspect-[500 / 341]",
    }.freeze
```
and concat `variant_aspect_ratio` to the class list in the template?

Reply:
Oh, yes! Great idea.